tests/test_game_logic.py

```
@@ -92,61 +92,51 @@ def test_create_grid_returns_deep_copy():
-    locked = empty_grid()
-    grid = create_grid(locked)
-
-    # 左に移動できることを確認
+    # 正常な移動: 左に移動できることを確認
     mino = DummyMino(5, 5, O)
     _, moved_mino = keyOperation(True, pygame.K_LEFT, grid, mino)
     assert moved_mino.x == 4
 
-    # 左端では移動できないことを確認
+    # 左端制限: 左端では移動できないことを確認
     # Oミノは相対位置(2,1)から始まるので、x=-1が実質的な左端
```

Ruff (0.14.10)

Comment contains ambiguous `ノ` (KATAKANA LETTER NO). Did you mean `/` (SOLIDUS)?

(RUF003)